README.md

```diff
@@ -142,6 +142,26 @@ Enables tables using pipes as in [GitHub Flavored Markdown][gfm-tables].
 
 Use class `TablesExtension` in artifact `commonmark-ext-gfm-tables`.
 
+### Metadata
+
+Enables metadata block in YAML format. This extension only supports limited features. The example is following:
```

Comment:
Replace "This extension only supports limited features" with "This extension only supports a subset of YAML syntax"? And replace "The example is following:" with "Here's an example of what's supported:" to make it clear that this is it.